proof/refine/ARM/Ipc_R.thy

```diff
@@ -5011,6 +5067,17 @@ crunches replyPush
   and valid_queues'[wp]: valid_queues'
   (wp: crunch_wps hoare_vcg_all_lift valid_irq_node_lift simp: crunch_simps valid_mdb'_def)
 
+lemma valid_tcb_state'_update[simp]:
+  "valid_tcb_state' ts (s\<lparr>ksReadyQueues := qs\<rparr>) = valid_tcb_state' ts s"
+  "valid_tcb_state' ts (s\<lparr>ksReadyQueuesL1Bitmap := l1b\<rparr>) = valid_tcb_state' ts s"
+  "valid_tcb_state' ts (s\<lparr>ksReadyQueuesL2Bitmap := l2b\<rparr>) = valid_tcb_state' ts s"
+  by (auto simp: valid_tcb_state'_def valid_bound_obj'_def split: thread_state.splits option.splits)
+(* ryanb - remove if unneeded
```

Comment:
Just a small nitpick. Is this meant to be removed? And, I feel like the equalities above exist somewhere, but i am not sure.
Other than this, this looks great, ready to go.

Reply:
Yep, was planning to remove this once the tests pass assuming there were no other comments.